lms/resources/_js_config.py

```diff
@@ -48,6 +48,9 @@ def config(self):
             # Our JSON-RPC server passes this to the Hypothesis client over
             # postMessage.
             "hypothesisClient": self._hypothesis_client,
+            # The LMS name to use in user-facing messages.
+            # Shown on the "Select PDF from Canvas" button label.
+            "lmsName": "Canvas",
```

Comment:
This is super weird, because a lot of the time this is wrong. Previously presumably this wouldn't have been the case as it was only added in a Canvas files context?

We don't happen to have a string around in the LTI parameters that would actually tell is the LMS name do we?